synapse/tests/test_lib_stormtypes.py

```diff
@@ -207,6 +207,30 @@ async def test_storm_lib_base(self):
             self.eq(erfo[1][0], 'NoSuchName')
             self.eq(erfo[1][1].get('name'), 'newp')
 
+            # lib.len()
+            opts = {
+                'vars': {
+                    'true': True,
+                    'list': [1, 2, 3],
+                    'dict': {'k1': 'v1', 'k2': 'v2'},
+                    'str': '1138',
+                    'bytes': b'o'
+                }
+            }
+
+            msgs = await core.stormlist('$lib.print($lib.len($list))', opts=opts)
```

Comment:
`self.eq(3, await core.callStorm('return($lib.len($list))')`

Reply:
scraping text output is less reliable than returning an explicit value

Reply:
updated